README.md

```diff
@@ -49,6 +49,14 @@ To start the local development server:
 
 And fire up __[http://localhost:9000](http://localhost:9000)__!
 
+To have the URL open after the server starts:
+
+    OPEN=1 npm start
+
+To enable live reloading (i.e., automatic page refreshing upon any file changes):
+
+    OPEN=1 LIVE=1 npm start
+
```

Comment:
Another approach is to use -- and then traditional CLI args that get parsed by minimist. Eg:

`npm start -- -l -o`